include/cpu/apic.h

```diff
@@ -21,7 +21,7 @@
 
 #include <cpu/config.h>
 
-#if defined(TARGET_I386)
+#if defined(TARGET_I386) || defined(TARGET_X86_64)
```

Comment:
Is this necessary? AFAIK, TARGET_X86_64 implies TARGET_I386.
In anycase, this should be sent to a separate PR.